org.ghidra_sre.Desktop.json

```diff
@@ -0,0 +1,72 @@
+{
+    "app-id" : "org.ghidra_sre.Desktop",
```

Comment:
I think I'd prefer `org.ghidra_sre.Ghidra` but it's a nitpick, feel free to ignore it.

Reply:
https://github.com/NationalSecurityAgency/ghidra/issues/2739